frontend/packages/console-dynamic-plugin-sdk/src/app/components/status/Status.tsx

```diff
@@ -71,6 +74,7 @@ const Status: React.FC<StatusProps> = ({
     case 'Not Ready':
     case 'Cancelling':
     case 'Terminating':
+    case 'Uninstalling':
       return <StatusIconAndText {...statusProps} icon={<BanIcon />} />;
```

Comment:
Can you add "Uninstalled" status as well?

Reply:
`export const OtherReleaseStatuses = ['unknown', 'uninstalled', 'superseded', 'uninstalling'];`

In `OtherReleaseStatuses`, we added `uninstalled` as well. So I thought better to add it.

Let's not consider if it is not necessary.